### PR TITLE
Add prometheus-based aggregated api call latency graph

### PR DESCRIPTION
**Pull request by mm4tt**

### CODE REVIEW

**Comment by mm4tt** [comment]
/assign @oxddr 

This is graph of the exact query we use to evaluate Prometheus api-call-latency SLO.
I found this useful when working on golang1.13 verification as this metric is much more sensitive to spikes - it catches them very well. Having this graph on top of the graph that roughly calculates the old api-call-latency SLI should help us understand the differences between them.

File: clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py [hunk 1]
```diff
@@ -58,6 +58,12 @@ def simple_graph(title, exprs, yAxes=None, legend="", interval="5s"):
         g.single_y_axis(format=g.SECONDS_FORMAT),
         "{{verb}} {{scope}}/{{resource}}",
     ),
+    simple_graph(
+        "API call latency aggregated (prometheus)",
+        'quantile_over_time(0.99, apiserver:apiserver_request_latency:histogram_quantile{verb!="LIST", verb!="WATCH", verb!="CONNECT", verb!="DELETECOLLECTION"}[5d])',
```
Comment (oxddr) [inline clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py]: Why 5d? For our tests 12h should be sufficient.

[the remark this answers is not in the document]
Comment (oxddr) [reply clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py]: Actually, why do you exclude LISTs from here? It seems to be a regular API call we want to track.

[the remark this answers is not in the document]
Comment (mm4tt) [reply clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py]: Done, there is a different threshold for LIST and this graph is only for call with 1s threshold (updated the graph title).

**Comment by mm4tt** [comment]
@oxddr, PTAL

File: clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (oxddr) [reply clusterloader2/pkg/prometheus/manifests/dashboards/master-dashboard.dashboard.py]: Make sense.

**Pull request merged**